node/index.d.ts

```diff
@@ -1076,6 +1076,40 @@ declare module "zlib" {
     export function unzip(buf: Buffer, callback: (error: Error, result: Buffer) => void): void;
     export function unzipSync(buf: Buffer, options?: ZlibOptions): Buffer;
 
+    export var constants: {
```

Comment:
In the description of the nodejs/node PR the author says
`refactor zlib constants for consistency with the changes to constants in fs, os`

the os module was declared directly below `zlib`, so I copied it from there. But I think you're right. I'll do it again and copy it from fs-module declaration.